packages/aws-client/src/KMS.ts

```diff
@@ -23,12 +23,12 @@ export const createKey = (params: AWS.KMS.CreateKeyRequest = {}) =>
  * @param {string} Plaintext - the string to be encrypted
  * @returns {Promise<string>} the Base 64 encoding of the encrypted value
  */
-export const encrypt = async (KeyId: string, Plaintext: string) => {
-  const { CiphertextBlob } = await kms().encrypt({ KeyId, Plaintext }).promise();
+export const encrypt = async (KeyId: string, Plaintext: Uint8Array) => {
```

Comment:
Yikes - this is not-insignificant change if true, right?  

At a minimum this needs a changelog entry, but we should probably also make sure our upstream code that is .js is doing the right thing here.   

e.g.

```
> textstring = new TextEncoder().encode('foo')
Uint8Array(3) [ 102, 111, 111 ]
> new TextDecoder().decode(textstring)
'foo'
``````

I may be misremembering how the varying string types work, but this needs an explination/CL entires/etc before merging.

Reply:
If the behavior of kms.encrypt changed, we should probably keep this method signature the same and convert the string, however we need to be *very careful* this wasn't accidently mistyped and we haven't been doing the right thing here.

Reply:
Hmm, the unit doesn't indicate an issue, though, which leads me to wonder if the typing is incorrect?    What happens if we explicitly convert the string?